modules/seqexec/server/src/main/scala/seqexec/server/InstrumentSystem.scala

```diff
@@ -20,7 +21,7 @@ trait InstrumentSystem[F[_]] extends System[F] with InstrumentGuide {
   val observeControl: InstrumentSystem.ObserveControl[F]
 
   def observe(
-      config: Config): SeqObserveF[F, ImageFileId, ObserveCommandResult]
+      config: Config): Kleisli[F, ImageFileId, ObserveCommandResult]
```

Comment:
This was suggested by @tpolecat as a good replacement for the previous construct